FileNameToURL should return utf8.

Currently, URLToFileName() and FileNameToURL() are not symmetric.
URLToFileName() will assume input url is utf8 encoded (because wxURI::Unescape).
But FileNameToURL() will not encode unicode path as utf8.
diff --git a/wxWidgets/src/common/filesys.cpp b/wxWidgets/src/common/filesys.cpp
--- a/wxWidgets/src/common/filesys.cpp
+++ b/wxWidgets/src/common/filesys.cpp
@@ -689,17 +689,17 @@
     // doesn't accept "file:///c%3a/foo.bar" -- but then we never made any
     // guarantees about general suitability of the strings returned by this
     // method, they must work with wxFileSystem only and not encoding the colon
     // breaks handling of "http://wherever/whatever.zip#zip:filename.ext" URLs
     // so we really can't do this without heavy changes to the parsing code
     // here, in particular in GetRightLocation()
     url.Replace(wxT(":"), wxT("%3A"));
     url = wxT("file:") + url;
-    return url;
+    return wxURI(url).BuildURI();
 }
 
 
 ///// Module:
 
 class wxFileSystemModule : public wxModule
 {
     DECLARE_DYNAMIC_CLASS(wxFileSystemModule)
